### PR TITLE
Added documentation for CP-45 and CP-89

### PR DESCRIPTION
**Pull request by b0bkaT**

### CODE REVIEW

File: releases/0.8.0/index.md [hunk 1]
```diff
@@ -18,14 +18,16 @@ custom_css: releases
 *(These are changes to ontologies, classes or properties in the preexisting ontology that make the new release non-backward-compatible.)*
 
 * Restructured repository by moving previously prefixed uco-* ontology files under new directory /ontology/* ([*Change Proposal 56*](https://drive.google.com/file/d/1PCjdCGw7wgFnPfbsCn0Fdvdnq9NDW6yA/view))
+* Remove ActionReferenceFacet and move properties into action:Action ([*Change Proposal 89*](https://drive.google.com/file/d/1njOUmpNNCTolnGkXoKv5ext-ZK18108u/view))
```
Comment (balon) [inline releases/0.8.0/index.md]: The document attached is missing the implementation completion date, you may want to update this to the date of the PR merge then re-upload the PDF. I am unsure if the link will change.

File: releases/0.8.0/index.md [hunk 2]
```diff
@@ -18,14 +18,16 @@ custom_css: releases
...
 
 #### Changes
 *(These are general changes to the preexisting ontology that are not breaking or range changes.)*
 
+* Remove observable:fileSystemType from FileFacet ([*Change Proposal 45*](https://drive.google.com/file/d/1ycbboFe8wEYzD3AD6lbUadwXXFuGF3Dl/view))
```
Comment (balon) [inline releases/0.8.0/index.md]: This link is either not set to 'view all with link' or the document does not exist.

**Review by balon — CHANGES_REQUESTED** [review]
Github is reporting a merge conflict between the line which describes CP-97. This may be a symptom of the branch starting from before CP-97 was merged into master.

**Comment by b0bkaT** [comment]
Nuking this branch and including these updates in a different one.

**Pull request closed without merging**